sdcm/cluster_k8s/__init__.py

```diff
@@ -93,6 +93,7 @@
 SCYLLA_MANAGER_NAMESPACE = "scylla-manager-system"
 SCYLLA_NAMESPACE = "scylla"
 MINIO_NAMESPACE = "minio"
+SCYLLA_CONFIG_NAME = "scylla-config"
```

Comment:
I would call it DEFAULT_SCYLLA_CONFIGMAP_NAME
Since it could be changed in s-o config

Reply:
It is not "default" it is what we specify in ScyllaCluster CRD as scylla config file. It must be used by S-O as is. And if not, it will be bug.